env.sh-template

```diff
@@ -15,7 +15,11 @@ LOCAL_JWT_SIGNING=TRUE
 DOCKER_COMPOSE=TRUE
 #DOCKER_COMPOSE=FALSE
 
-DOCKER_IMAGE="digitalasset/daml-sdk:1.5.0"
+# OCSP Checking
+OCSP_CHECKING=""
+#OCSP_CHECKING="--cert-revocation-checking true"
```

Comment:
why not use the one you commented out?

Reply:
I have two modes of operation and this allows me to switch between them. My general usecase is not to enable this. I plan to write a short note on certificate revocation to support this but wanted to get the sample out first.